### PR TITLE
Add the missing output variable

### PR DESCRIPTION
**Pull request by harikt**
| Q | A |
| --- | --- |
| Doc fix? | [yes] |
| New docs? | [no] |
| Applies to | [2.5, 2.6] |
| Fixed tickets | [NIL] |

For the 2.3 branch there is no `table.rst` file as it is introduced in 2.5 .

I have checkout from 2.5 and created the PR .

Let me know if I need to send this to master for 2.5 is not LTS .

### CODE REVIEW

File: components/console/helpers/table.rst [hunk 1]
```diff
@@ -25,6 +25,9 @@ To display a table, use :class:`Symfony\\Component\\Console\\Helper\\Table`,
 set the headers, set the rows and then render the table::
 
     use Symfony\Component\Console\Helper\Table;
+    use Symfony\Component\Console\Output\ConsoleOutput;
```
Comment (xabbuh) [inline components/console/helpers/table.rst]: I would move this befor the existing `use` statement.

**Comment by xabbuh** [comment]
@harikt Thanks for your pull request. `2.5` was exactly the right target branch as Symfony 2.5 is maintained until the end of this month.

File: components/console/helpers/table.rst (hunk 1 above)
Comment (harikt) [inline components/console/helpers/table.rst]: @xabbuh not sure I understood you. What is the benefit ?

Comment (xabbuh) [inline components/console/helpers/table.rst]: sorry for the confusion, it's right the way it is

**Comment by WouterJ** [comment]
I'm -1 for this change honestly. This code demostrates the table helper. That means you'll always use it in one of the `execute`/`interact`/... methods of the Command, which means you'll have access to the `$output` parameter of those methods.

What's missing instead imo is a simple `// ...`, which will indicate that something is missing (the class + method). I'm afraid that the current code block would confuse people and makes people instantiating their own `$output` instead of using the parameter one.

**Comment by harikt** [comment]
@WouterJ for you as an experienced symfonian it will be easy and may not find value in it. But if you ask from someone you may get some feedback :) .

**Comment by WouterJ** [comment]
@harikt I'm aware that this can be confusing. What about adding the method and class definitions?

````diff
  use Symfony\Component\Console\Helper\Table;
+ // ...
+
+ class BookCommand extends Command
+ {
+     public function execute(InputInterface $input, OutputInterface $output)
+     {
- $table = new Table($output);
+         $table = new Table($output);
// ... the rest of the code
+     }
+ }
````

**Comment by harikt** [comment]
@WouterJ when you can run without the command I wonder why you need to show like that. What is the advantage you are seeing ?

**Comment by WouterJ** [comment]
@harikt the complete Symfony Console component is designed the way that you use this logic inside `Command#execute` or `Command#interact`. This is the official way we document, how would you use this helper?

**Comment by weaverryan** [comment]
Hi @harikt!

I actually agree with @WouterJ - the "normal" use for this is inside the `execute(InputInterface $input, OutputInterface $output) {}` of a command, so the `$output` you're using is passed to you as an argument. However, you *do* technically bring up a valid usage - you could *just* use the `Table` class inside a flat PHP script (not inside a Console component `*Command` class) and create a `ConsoleOutput` to render things. Was this your use-case?

If it is, it's not the main usage, so if we show the creation of the `ConsoleOutput` being done, that will actually be incorrect for most users. I wonder if @WouterJ's suggested change above would help - at least for more advanced use-cases, you'd realize that you are *not* in a Command and may need to create an Output manually.

Thanks!

**Comment by harikt** [comment]
I am ok with the comments https://github.com/symfony/symfony-docs/pull/4812#issuecomment-70278309

**Comment by xabbuh** [comment]
@harikt Cool, will you be able to make those changes? I think once that's done, the PR is ready to be merged.

**Comment by WouterJ** [comment]
Hi @harikt! There is a [doc sprint day](http://symfony.com/blog/symfony-docs-hack-day-needs-you-on-may-23rd) tomorrow. I would like to have this PR finished at the end of tomorrow, can you work on it or should we put it on the list so someone else can finish it?

**Comment by harikt** [comment]
Hey @WouterJ , 

You can assign this to someone else. My internet is broken and have very limited access. Else I wished I could. Also got busy with family and things :( .

Enjoy the sprint.

**Comment by harikt** [comment]
Hey @xabbuh , @WouterJ , 

if no one has taken this. I will look into this in the weekend. ( tomorrow / day after tomorrow ) . How about that ?

**Comment by harikt** [comment]
I also noticed one thing . From the https://symfony.com/components there is no link to Validation docs. Why is it not added? Is there any other tickets already ?

**Comment by xabbuh** [comment]
@harikt This sounds good.

> From the https://symfony.com/components there is no link to Validation docs. Why is it not added? Is there any other tickets already ?

They don't exist yet (see #958).

**Comment by harikt** [comment]
@xabbuh I forgot to ask a question. To which branch it should go ?

**Comment by harikt** [comment]
I send a PR to 2.7 as 

@xabbuh as already mentioned the time frame 

> Thanks for your pull request. 2.5 was exactly the right target branch as Symfony 2.5 is maintained until the end of this month.

**Comment by WouterJ** [comment]
Thanksfor your PR, @harikt! I'm going to close this one, as it's replaced by your new one.

**Pull request closed without merging**